### PR TITLE
APPTRACK-827 Banner Notifying an Applicant While They are Editing a submitted Application That Their Changes Will Not Save

### PR DESCRIPTION
**Pull request by lfcabrera14**
APPTRACK-827 Banner Notifying an Applicant While They are Editing a submitted Application That Their Changes Will Not Save

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature
Comment (Mariachaudhry) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: Are the bottom commented ticket numbers needed?

Comment (katm007) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: There are commented out lines on 268-269. Please remove them if they're not needed.

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java
Comment (katm007) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java:1]: A lot of methods here don't have doc strings.

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: @Mariachaudhry yes they are needed.
@katm007 un commented those steps

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java:1]: I will add them to it

**Review by Mariachaudhry — APPROVED** [review]

**Review by katm007 — APPROVED** [review]
approved

**Pull request merged**